site/_docs/reference.md

```diff
@@ -2775,6 +2775,17 @@ BigQuery's type system uses confusingly different names for types and functions:
   function, return a Calcite `TIMESTAMP WITH LOCAL TIME ZONE`;
 * Similarly, `DATETIME(string)` returns a Calcite `TIMESTAMP`.
 
+#### ClickHouse some functions must be camel case format, comparisons are as follows:
```

Comment:
That's not what I had in mind, but I see your point.
I guess more work needs to be done to support these properly:
- these names need to be added to the correct operator table
- internally these functions are represented using the existing functions STARTS_WITH, etc.
- on deparsing the correct name is used

I apologize for the confusion I created, let's remove this list from the documentation and merge the PR as it was before.

Reply:
OK, I had remove infomations from reference.md. @mihaibudiu @NobiGo